cypress/e2e/spec.cy.ts

```diff
@@ -62,255 +63,185 @@ describe('Encrypt and Decrypt String', () => {
   });
 
   it('authSig is saved', () => {
-    // window = await cy.window();
     // expect saveParams not empty
     expect(savedParams.authSig).to.be.an('object');
   });
 
   it('connect lit node client', () => {
-    cy.window().then(async (window) => {
+    cy.window().then(async () => {
       const client = new LitJsSdk.LitNodeClient({ litNetwork: 'serrano' });
       await client.connect();
       savedParams.litNodeClient = client;
       expect(client.config.litNetwork).to.be.eq('serrano');
     });
   });
+});
 
-  it('encrypts string', async () => {
-    const res = await LitJsSdk.encryptString('This test is working! Omg!');
-    savedParams.encryptedString = res.encryptedString;
-    savedParams.symmetricKey = res.symmetricKey;
-    expect(savedParams.encryptedString).to.be.a('Blob');
-    expect(savedParams.symmetricKey).to.be.a('Uint8Array');
-  });
-
-  it('turns blob to base64 string', async () => {
-    const base64 = await LitJsSdk.blobToBase64String(
-      savedParams.encryptedString
-    );
-    savedParams.encryptedString = base64;
-    expect(savedParams.encryptedString).to.be.a('string');
-  });
-
-  it('saves encryption key', async () => {
-    // const { encryptedString, symmetricKey } = savedParams;
-
-    const encryptedSymmetricKey =
-      await savedParams.litNodeClient.saveEncryptionKey({
+describe('Encrypt and Decrypt', () => {
+  it('encrypts', async () => {
+    cy.window().then(async () => {
```

Comment:
I'm not sure why, but without wrapping each functionality with this `cy.window()` the `wasm` is undefined and not initialized